@here/harp-omv-datasource/lib/OmvTileInfoEmitter.ts

```diff
@@ -66,9 +66,17 @@ export class OmvTileInfoEmitter implements IOmvEmitter {
             const infoTileTechniqueIndex = tileInfoWriter.addTechnique(technique);
 
             for (const pos of geometry) {
-                const { x, y } = targetProjection
-                    .reprojectPoint(webMercatorProjection, pos, worldPos)
-                    .sub(center);
+                const worldPos = tile2world(
```

Comment:
again, local function, or a class instance which is initialized with some constant values once, and then just calls projectAndAdd(xxx) ? hhhmmhh

Reply:
Done.